pkg/controller/accountclaim/accountclaim_controller.go

```diff
@@ -113,6 +124,65 @@ func (r *ReconcileAccountClaim) Reconcile(request reconcile.Request) (reconcile.
 		return reconcile.Result{}, nil
 	}
 
+	if accountClaim.Spec.BYOC {
+
+		if accountClaim.Spec.AccountLink == "" {
+
+			//Create a new account with BYOC flag
+			newAccount := utils.NewAccountForCR(awsv1alpha1.AccountCrNamespace)
+			newAccount.Spec.BYOC = true
+			newAccount.Spec.AwsAccountID = accountClaim.Spec.BYOCAWSAccountID
+			newAccount.Spec.ClaimLink = accountClaim.ObjectMeta.Name
+			newAccount.Spec.ClaimLinkNamespace = accountClaim.ObjectMeta.Namespace
```

Comment:
helper func to update everything in Spec?

Reply:
:+1: